doc/src/usage.md

```diff
@@ -32,6 +32,13 @@ target, see [Per Target options](#per-target-options) for details.
 [`INTERFACE`]: https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries
 [target_link_libraries]: https://cmake.org/cmake/help/latest/command/target_link_libraries.html
 
+### Install crate and headers with `corrosion_install`
+
+The default CMake [install commands] do not work correctly with the targets exported from `corrosion_import_crate()`.
```

Comment:
I think in the anchor thats imported, you define [`install(TARGETS)`] but  not [install commands].

Reply:
Fixed.